tests/browsing_context/capture_screenshot/test_capture_screenshot.py

```diff
@@ -68,6 +68,50 @@ async def test_screenshot(websocket, context_id, png_filename,
         assert_images_equal(resp["result"]["data"], png_base64)
 
 
+@pytest.mark.asyncio
+async def test_screenshot_element(websocket, context_id, query_selector,
+                                  get_cdp_session_id):
+    await goto_url(websocket, context_id, 'data:text/html,<div>hello</div>')
+    session_id = await get_cdp_session_id(context_id)
+
+    # Set a fixed viewport to make the test deterministic.
+    await execute_command(
```

Comment:
There is BiDi command to set viewport that one is preferred if there are no issue with it.

Reply:
Comment not addressed.

Reply:
+1

This pattern was used before there was a native BiDi command, but now that there's one, it would be better to use it directly.

Bonus points if you refactor all existing usages as well (there shouldn't be many of them). This could be done in another PR.

Reply:
I think it makes sense to do this in another PR if we have multiple usage.